governance/angle-dao.md

```diff
@@ -23,30 +23,20 @@ In particular, it can make the following changes:
 - Deploy new stablecoins
 - Deploy/revoke new strategies to get yield and choose in which proportions to distribute this yield to SLPs
 - Deploy/revoke collateral types
-- Add/remove incentives contracts for stable holders, SLPs and HAs
+- Add/remove incentives on staking contracts
 - Upgrade oracles and other contracts throughout the system
-- Change bonding curve parameters
 - Deploy the surplus of the protocol
```

Comment:
We should add veANGLE advantages here: vote for gauge weights (which is clearer than add:remove incentives on staking contracts)